controllers/image_generator.go

```diff
@@ -73,87 +53,148 @@ const (
 	peerpodsCMAWSImageKey    = "PODVM_AMI_ID"
 	peerpodsCMAzureImageKey  = "AZURE_IMAGE_ID"
 	fipsCMKey                = "BOOT_FIPS"
-	defaultVMType            = "VM"
+	procFIPS                 = "/proc/sys/crypto/fips_enabled"
+	AWSProvider              = "aws"
+	AzureProvider            = "azure"
```

Comment:
nit, do you mind adding as part of the series also "peerpodsImageJobsPathLocation" and the other image creation related consts, i think it would make more sense